### ISSUE TITLE
Ability to import bookmarks from Safari

### ISSUE THREAD

**Issue by bbondy** [brave/browser-laptop#1004]
Safari exports are not importing currently

**Comment by bradleyrichter** [comment]
Mine imported to the about: bookmarks page but did not appear in the toolbar.

They can be dragged to the toolbar and used though.

**Comment by bbondy** [comment]
where did you have them in safari?

**Comment by bradleyrichter** [comment]
Jim Marshall Wednesday 00:35 
Same problem with 0.9.1 update.

 
Jim Marshall Apr 06 22:06 
I just downloaded Brave. I exported my Safari bookmarks in html and tried to import them into Brave. AS far as I can tell, the only ones that were imported were the favorites, which I am able to display in the Bookmarks toolbar. However, I cannot find or see any others. The “otherBookmarks” folder in the Bookmarks Manager… is empty.

I checked the exported html file and all the bookmarks are there, so this appears to be a problem with Brave importing everything other than the favorites, ie almost all my bookmarks.

I did close and reopen Brave, and tried to import the html a second time, all to no avail.
